src/support/testUtils.tsx

```diff
@@ -23,16 +44,22 @@ export const testUser = {
   emailVerified: true,
   getIdtoken: () =>
     new Promise<string>((resolve, _reject) => resolve('xxxxxxx')),
-} as User
+} as unknown as User
```

Comment:
For some reason just recently TypeScript starting whining about this when it never did before. Maybe because of the upgrade?